dali/operators/signal/decibel/to_decibels_op_gpu.cu

```diff
@@ -33,6 +33,7 @@ class ToDecibelsImpl : public OpImplBase<GPUBackend> {
 
   explicit ToDecibelsImpl(ToDecibelsArgs args)
       : args_(std::move(args)) {
+    max_out_.SetContiguity(BatchContiguity::Contiguous);
```

Comment:
This line seems to be disturbing. Does that mean, that it's possible that the change breaks code of existing operators?

Reply:
Not really. To get access to a contiguous allocation, you need to use one of the API that will throw you an error if you don't meet the preconditions and your allocation is not contiguous. So it won't break silently, on the first use you will get the error, hence why this was adjusted.